src/components/Header.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+
+export const Header = () => {
+    const [isDarkMode, setIsDarkMode] = useState(false);
+
+    const toggleTheme = () => {
+        setIsDarkMode(!isDarkMode);
+
+        if (!isDarkMode) {
+            document.body.classList.add("dark-theme");
+        } else {
```

Comment:
You can take advantage of the second argument of classlist like this : 

`document.body.classList.toggle("dark-theme", !isDarkMode);`